docs/en/using-status/leave-a-group-chat.md

```diff
@@ -8,7 +8,31 @@ hide:
 
 # Leave a group chat
 
- :octicons-tools-24:
+If you don’t want to be a member of a [group chat][understand-group-chats] anymore, you can leave the chat. You can always [re-join the chat][join-a-group-chat] later if you change your mind.
 
-!!! note ""
-     We're working on this content.
+Once you leave, a note appears in the group chat's message history letting other members know that you left the chat. You can still see the previous message history for a group chat you left.
+
+If someone is adding you to group chats you don't want to be in, consider [changing who can invite you to group chats][change-who-can-invite-you-to-group-chats] or [blocking that person][block-someone-in-status].
+
+!!! note
+	The group chat owner can't leave without deleting the chat for everyone. For more information, check out [Delete a group chat][delete-a-group-chat].
+
+## Leave a group chat
+
+=== "Mobile"
+
+    1. From the tab bar, tap :mobile-messages: Messages.
```

Comment:
This final **Messages** in step (1) goes in bold typeface, even when the text doesn't appear on the screen.